nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -393,7 +401,9 @@ def forward(self, audio_signal, length, cache_last_channel=None, cache_last_time
             audio_signal, pos_emb = self.pos_enc(x=audio_signal)
 
         # pad_mask is the masking to be used to ignore paddings
-        pad_mask = self.make_pad_mask(max_audio_length=max_audio_length, seq_lens=padding_length)
+        pad_mask = torch.arange(0, max_audio_length, device=audio_signal.device).expand(
+            padding_length.size(0), -1
```

Comment:
Why not use the utility method. Much safer for more use cases

Reply:
That method was just a simple single line and it was just used once. It may have been used before multiple places but now it is just here.

Reply:
The utility being a single line means subclasses have the opportunity to change it to something else. That's better in the long term.

Reply:
reverted it back.

Reply:
It's not reverted in the pr